src/Microsoft.ML.CpuMath/SseIntrinsics.cs

```diff
@@ -20,7 +20,7 @@
 
 namespace Microsoft.ML.Runtime.Internal.CpuMath
 {
-    internal static class SseIntrinsics
+    public static class SseIntrinsics
```

Comment:
Why is this public?  I don't think it should be.

Reply:
`SseIntrinsics` has been made public since I wanted to call `SseIntrinsics` methods directly from another assembly in perf tests.  Since now the original `CpuMathUtils` code path mostly picks up the `AVX` intrinsics, I may need to call `SseIntrinsics` methods directly to get the `SSE` intrinsics.

Reply:
IMO - We should [InternalsVisibleTo](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.internalsvisibletoattribute) for things like this.